client/src/includes/breadcrumbs.test.js

```diff
@@ -0,0 +1,35 @@
+import initCollapsibleBreadcrumbs from './breadcrumbs';
+
+describe('initCollapsibleBreadcrumbs', () => {
+  it('should do nothing if there is no breadcrumbs container', () => {
+    jest.spyOn(document, 'addEventListener').mockImplementation(() => {});
```

Comment:
Move this soon statement out to under the describe block - jest spyon is a bit confusing but it runs for the test as a whole - wherever you put it. 

To avoid confusion - best to put this in the parent describe block or above it. 


Note that spyOn does not change the behaviour of the function - just let's you check it has been called.

Reply:
Hopefull resolved